CONTRIBUTING.md

```diff
@@ -62,6 +62,15 @@ On another terminal, move to the war folder and start a [webpack](https://webpac
 cd war; yarn start
 ```
 
+### Gitpod
+
+You can open this project as a [Gitpod workspace](https://www.gitpod.io/) which comes pre-configured with all the tools you will need.
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/jenkinsci/design-library-plugin)
```

Comment:
```suggestion
[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/jenkinsci/jenkins)
```
👀

Reply:
Actually, doesn't that open the upstream repository? You'd need to open your forked repository to propose changes.

Reply:
No that would start you from some super outdated fork probably.

The normal workflow is to start from upstream/master, then create branch off of that, push to your fork and create a PR from it.

So when you get there in gitpod using the above button you will just need to run `git switch -c my-branch`

Reply:
Oic, thanks for the clarification.